bundles/framework/coordinatetool/plugin/CoordinatePluginHandler.js

```diff
@@ -19,27 +17,30 @@ class UIHandler extends StateHandler {
         this.reverseGeocodingIds = this.config.reverseGeocodingIds?.split(',');
         this.setState({
             xy: {},
-            displayXy: {},
+            latField: '',
+            lonField: '',
             loading: false,
-            popupControls: null,
             showMouseCoordinates: false,
             selectedProjection: this.originalProjection,
             emergencyInfo: null,
             reverseGeocodeNotImplementedError: false,
             reverseGeoCode: [],
-            showReverseGeoCode: false
+            showReverseGeoCode: false,
+            approxValue: false
         });
         this.popupControls = null;
         this.eventHandlers = this.createEventHandlers();
         this.decimalSeparator = Oskari.getDecimalSeparator();
         this.preciseTransform = Array.isArray(this.config.supportedProjections);
-        this.updateLonLat();
+        this.updateLonLat(this.getMapXY(), true, true, true);
         this.getEmergencyCallInfo(this.state.xy);
```

Comment:
updateLonLat() seems to call getEmergencyCallInfo() so we don't need another call here.